### ISSUE TITLE
❓ [Question]Why the GPU memory has doubled  when I loaded model from Torch-TensorRT by Pytorch? 

### ISSUE THREAD

**Issue by Jancapcc** [pytorch/TensorRT#1189]
## ❓ Question

When I'm using Pytorch to load model from Torch-TensorRT(torch.jit.load (*.ts)) file, the model's GPU memory has doubled(1602MB to 3242MB of GPU Memory from Nvidia-smi). At the same time, the gradient of model tensors are both not included. What I'm concern is that the context memory of torch is not reused, is restart a new context memory of torch.  

## Environment

> Build information about Torch-TensorRT can be found by turning on debug messages

 - PyTorch Version (e.g., 1.0):1.10.0
 - OS (e.g., Linux): Linux
 - How you installed PyTorch (`conda`, `pip`, `libtorch`, source): pip
 - Python version: 3.7
 - CUDA version:11.2
 - Any other relevant information: torch-tensorrt version: 1.1.0
- NVIDIA GPU: Tesla v100

## Additional context

import torch
import torch_tensorrt

# memory is 1.6G
a= torch.randn()
a= torch.randn([1,1,224,224])
a.cuda()

# memory become 3.2G
model = torch.jit.load()
